addons/hr_work_entry_holidays/models/hr_contract.py

```diff
@@ -77,81 +63,3 @@ def _get_sub_leave_domain(self):
             domain,
             [('holiday_id.employee_id', 'in', self.employee_id.ids)] # see https://github.com/odoo/enterprise/pull/15091
         ])
```

Comment:
Hi @robcoekaerts ,

The purpose of _get_sub_leave_domain is to include an additional domain check in the parent method in hr_work_entry_contract. If we transfer this code to our new module, it may not access the parent method unless hr_work_entry_contract is installed.
Thank you.